test/messages-test.js

```diff
@@ -1,40 +1,48 @@
+import Gists from 'gists'
+import GoogleSpreadsheet from 'google-spreadsheet'
 import async from 'async'
 import chai from 'chai'
 import sinon from 'sinon'
 let messages = require('../scripts/messages.js').messages
 
 
 describe('Google Docs and Gists Logging', function () {
-    it('Google Docs messaging', function () {
-        let message = {
-            "text": "+todo testing @test",
-            "user": {
-                "name": "test"
-            }
-        }
-        sinon.stub(messages, "sendMessage", function () {
-            return "0"
+    let save
+
+    beforeEach(function () {
+        save = null
+    })
+
+    it('Formatting message', function () {
+        save = sinon.stub(messages, 'getRoom').callsFake(function (roomId, callback) {
+            sinon.assert.calledOnce(save)
+            callback("testname")
         })
 
-        messages.sendMessage(message, "dest", function (inf) {})
+        messages.formatMessage({"text": "+todo @test this", "room": "testingName", "user": {"name": "test", "login": "test"}}, function (data) {
+        })
+        save.reset()
```

Comment:
I'm not really getting were/how the assertion happen here, with `save.reset()`? Would the tests fail if you do it before `messsages.formatMessage(...)`￼